tests/test_gcovr.py

```diff
@@ -254,8 +264,8 @@ def find_reference_files(output_pattern):
 
 
 @pytest.fixture(scope="module")
-def compiled(request, name):
```

Comment:
Not sure why this fixture even depends on `request`… that's usually only necessary when introspecting the test setup or when parametrized fixtures.

Reply:
I can check on this.

Reply:
Test still work without the fixture.